### DISCUSSION TITLE
force autoclose even if edited

### DISCUSSION THREAD

**Discussion by tommyjcarpenter** [renovatebot/renovate#29986] [Suggest an Idea]
We have a helm docs job that runs on helm repos on any PR, which updates the README with helm values changes (https://github.com/norwoodj/helm-docs). This runs even on renovate PRs if the PR changes the chart in a way that adds new helm values etc. It does this by pushing a commit like so:

<img width="1251" alt="Screenshot 2024-07-02 at 07 55 01" src="https://github.com/renovatebot/renovate/assets/9419125/670acaab-374f-428f-808d-d11fbc45b1b3">

However, that prevents renovate from closing prior PRs because they've all been helm-doc'd:

```
Autoclosing Skipped
This PR has been flagged for autoclosing. However, it is being skipped due to the branch being already modified. 
Please close/delete it manually or report a bug if you think this is in error.
```

I'd like a flag in `renovate.json` that says something like `forceAutoCloseEvenIfEdited` - I searched the docs and didnt see anything like this: https://docs.renovatebot.com/configuration-options/

**Comment by rarkins** [comment]
Utilize gitIgnoredAuthors and add the other bot's git author so that it's ignored

**Reply by tommyjcarpenter** [reply]
thank you!
